### PR TITLE
Traducción de Inline Assembly

### PR DESCRIPTION
**Pull request by eenti**
Aquí está la traducción de Inline Assembly.  Por favor no duden en dejarme saber si hay algún problema con el PR.

### CODE REVIEW

**Comment by elijahhampton** [comment]
Hello @eenti, sorry for the wait. I will take care of your PR tonight.

File: docs/assembly.rst [hunk 1]
```diff
@@ -1,43 +1,30 @@
...
 
-Example
+Ejemplo
 -------
 
-The following example provides library code to access the code of another contract and
-load it into a ``bytes`` variable. This is possible with "plain Solidity" too, by using
-``<address>.code``. But the point here is that reusable assembly libraries can enhance the
-Solidity language without a compiler change.
+El siguiente ejemplo proporciona código de la librería para acceder al código de otro contrato y cargarlo en una variable ``bytes``. Esto también es posible con "Solidity plano" usando ``<dirección>.code``. Pero el punto aquí es que las librerías de ensamblado reutilizables pueden mejorar Solidity sin un cambio en el compilador.
```
Comment (elijahhampton) [inline docs/assembly.rst]: ``address`` shouldn't be translated to dirección here as it is referring to a keyword in the language

File: README.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ Por otro lado, también puedes encontrar la versión más actualizada de toda la
 
 Moderadores:
 
-- DanielMart-tech
-- elijahhampton
-- mektigboy
+- [DanielMart-tech]([url](https://github.com/DanielMart-tech))
+- [elijahhampton]([url](https://github.com/elijahhampton))
```
Comment (elijahhampton) [inline README.md:16]: Thanks!

File: docs/assembly.rst (hunk 1 above)
Comment (eenti) [reply docs/assembly.rst]: Sorry for the late response on my side! Thanks for letting me know, I must have translated it without realizing it, solved here https://github.com/solidity-docs/es-spanish/pull/123/commits/eec309e9043d9640ea6ea3ca92131d713bbf4fe2

**Pull request merged**